browser/mapping.go

```diff
@@ -248,10 +247,12 @@ func mapElementHandle(vu moduleVU, eh api.ElementHandle) mapping {
 		"type":                   eh.Type,
 		"uncheck":                eh.Uncheck,
 		"waitForElementState":    eh.WaitForElementState,
-		"waitForSelector": func(selector string, opts goja.Value) *goja.Object {
-			eh := eh.WaitForSelector(selector, opts)
-			ehm := mapElementHandle(vu, eh)
-			return rt.ToValue(ehm).ToObject(rt)
```

Comment:
Why don't we need to use `rt.ToValue(ehm).ToObject(rt)`?

Reply:
Goja handles it automatically. We don't need to return a `goja.Object` or `value`. The current mapping layer implementation unnecessarily uses objects.